cli/pcluster/commands.py

```diff
@@ -153,7 +153,6 @@ def create(args):  # noqa: C901 FIXME!!!
             cfn_params["ResourcesS3Bucket"] = bucket_name
 
         LOGGER.info("Creating stack named: %s", stack_name)
-        LOGGER.debug(cfn_params)
```

Comment:
The duplicate logging of `cfn_params` is done on old line 171. I left that one because it's logged after adding any extra parameters passed on the command line.